[CI] testShrinkAfterRollover fails on windows

**Build scan**:
https://gradle-enterprise.elastic.co/s/yqntm6q4v3exo
**Repro line**:

**Reproduces locally?**:
no
**Applicable branches**:
7.x
**Failure history**:

**Failure excerpt**:
```
org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT > testShrinkAfterRollover FAILED
15:36:13     java.lang.AssertionError: the original index was deleted by the shrink action
15:36:13         at __randomizedtesting.SeedInfo.seed([1D9468EA73C10D43:B76B83F9A260BC4]:0)
15:36:13         at org.junit.Assert.fail(Assert.java:88)
15:36:13         at org.junit.Assert.assertTrue(Assert.java:41)
15:36:13         at org.junit.Assert.assertFalse(Assert.java:64)
15:36:13         at org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.lambda$testShrinkAfterRollover$8(TimeSeriesDataStreamsIT.java:102)
15:36:13         at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1016)
15:36:13         at org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.testShrinkAfterRollover(TimeSeriesDataStreamsIT.java:102)
15:36:36 REPRODUCE WITH: gradlew ':x-pack:plugin:ilm:qa:multi-node:javaRestTest' --tests "org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.testShrinkAfterRollover" -Dtests.seed=1D9468EA73C10D43 -Dtests.security.manager=true -Dtests.locale=ko-KR -Dtests.timezone=America/St_Thomas -Druntime.java=8
15:36:36 
15:36:36 Suite: Test class org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT
15:36:36   1> [2021-01-18T10:35:01,916][INFO ][o.e.x.i.TimeSeriesDataStreamsIT] [testShrinkAfterRollover] before test
15:36:36   1> [2021-01-18T10:35:01,918][INFO ][o.e.x.i.TimeSeriesDataStreamsIT] [testShrinkAfterRollover] initializing REST clients against [http://127.0.0.1:63297, http://[::1]:63298, http://127.0.0.1:63365, http://[::1]:63366, http://127.0.0.1:63433, http://[::1]:63434, http://127.0.0.1:63501, http://[::1]:63502]
15:36:36   1> [2021-01-18T10:35:02,301][INFO ][o.e.x.TimeSeriesRestDriver] [testShrinkAfterRollover] HTTP/1.1 201 Created
15:36:36   1> [2021-01-18T10:36:11,628][INFO ][o.e.x.i.TimeSeriesDataStreamsIT] [testShrinkAfterRollover] Clearing template [logs-template]
15:36:36   1> [2021-01-18T10:36:12,842][INFO ][o.e.x.i.TimeSeriesDataStreamsIT] [testShrinkAfterRollover] after test
15:36:36   2> REPRODUCE WITH: gradlew ':x-pack:plugin:ilm:qa:multi-node:javaRestTest' --tests "org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.testShrinkAfterRollover" -Dtests.seed=1D9468EA73C10D43 -Dtests.security.manager=true -Dtests.locale=ko-KR -Dtests.timezone=America/St_Thomas -Druntime.java=8  2> 
15:36:36   2> java.lang.AssertionError: the original index was deleted by the shrink action
15:36:36         at __randomizedtesting.SeedInfo.seed([1D9468EA73C10D43:B76B83F9A260BC4]:0)
15:36:36         at org.junit.Assert.fail(Assert.java:88)
15:36:36         at org.junit.Assert.assertTrue(Assert.java:41)
15:36:36         at org.junit.Assert.assertFalse(Assert.java:64)
15:36:36         at org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.lambda$testShrinkAfterRollover$8(TimeSeriesDataStreamsIT.java:102)
15:36:36         at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1016)
15:36:36         at org.elasticsearch.xpack.ilm.TimeSeriesDataStreamsIT.testShrinkAfterRollover(TimeSeriesDataStreamsIT.java:102)
```

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/TimeSeriesRestDriver.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/TimeSeriesRestDriver.java
@@ -171,7 +171,8 @@ public final class TimeSeriesRestDriver {
         Map<String, LifecycleAction> warmActions = new HashMap<>();
         warmActions.put(SetPriorityAction.NAME, new SetPriorityAction(50));
         warmActions.put(ForceMergeAction.NAME, new ForceMergeAction(1, null));
-        warmActions.put(AllocateAction.NAME, new AllocateAction(1, singletonMap("_name", "javaRestTest-1,javaRestTest-2"), null, null));
+        warmActions.put(AllocateAction.NAME, new AllocateAction(1, singletonMap("_name", "javaRestTest-0,javaRestTest-1,javaRestTest-2," +
+            "javaRestTest-3"), null, null));
         warmActions.put(ShrinkAction.NAME, new ShrinkAction(1));
         Map<String, LifecycleAction> coldActions = new HashMap<>();
         coldActions.put(SetPriorityAction.NAME, new SetPriorityAction(0));

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesDataStreamsIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesDataStreamsIT.java
@@ -229,7 +229,7 @@ public class TimeSeriesDataStreamsIT extends ESRestTestCase {
     private static Settings getLifecycleSettings(String policyName) {
         return Settings.builder()
             .put(LifecycleSettings.LIFECYCLE_NAME, policyName)
-            .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 3)
+            .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 2)
             .build();
     }
 }

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
@@ -110,7 +110,7 @@ public class TimeSeriesLifecycleActionsIT extends ESRestTestCase {
         String originalIndex = index + "-000001";
         String shrunkenOriginalIndex = ShrinkAction.SHRUNKEN_INDEX_PREFIX + originalIndex;
         String secondIndex = index + "-000002";
-        createIndexWithSettings(client(), originalIndex, alias, Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 4)
+        createIndexWithSettings(client(), originalIndex, alias, Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 2)
             .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
             .put("index.routing.allocation.include._name", "javaRestTest-0")
             .put(RolloverAction.LIFECYCLE_ROLLOVER_ALIAS, alias));

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
@@ -187,7 +187,7 @@ public class TimeSeriesLifecycleActionsIT extends ESRestTestCase {
         assertBusy(() -> {
             String failedStep = getFailedStepForIndex(index);
             assertThat(failedStep, equalTo(ShrinkStep.NAME));
-        });
+        }, 30, TimeUnit.SECONDS);
 
         // update policy to be correct
         createNewSingletonPolicy(client(), policy, "warm", new ShrinkAction(expectedFinalShards));
@@ -496,7 +496,7 @@ public class TimeSeriesLifecycleActionsIT extends ESRestTestCase {
             Map<String, Object> settings = getOnlyIndexSettings(client(), index);
             assertThat(settings.get(EngineConfig.INDEX_CODEC_SETTING.getKey()), equalTo(codec));
             assertThat(settings.get(IndexMetadata.INDEX_BLOCKS_WRITE_SETTING.getKey()), equalTo("true"));
-        });
+        }, 30, TimeUnit.SECONDS);
         expectThrows(ResponseException.class, () -> indexDocument(client(), index));
     }
 

